pkg/cli/config/config.go

```diff
@@ -24,13 +25,14 @@ func GetBackplaneConfigFile() string {
 
 // Get Backplane config default path
 func getConfiDefaultPath(fileName string) string {
-	configDir, err := os.UserConfigDir()
-
+	UserHomeDir, err := os.UserHomeDir()
 	if err != nil {
 		return fileName
 	}
 
-	return configDir + "/" + fileName
+	configFilePath := filepath.Join(UserHomeDir, ".config", fileName)
```

Comment:
Any chance we could point this to a folder out the gate?  Especially if we are going to be targeting multiple different environments?

So it would ideally be something like:

`$HOME/.config/backplane/config.json`
`$HOME/.config/backplane/config.stage.json`
`$HOME/.config/backplane/config.int.json`

While the stage and int json files could be excluded and would just be personal overrides, having them in a directory gives them all one place to live together, along with any other customizations that the end-user wants to make.

Reply:
Thanks for the suggestion, this really looks like an ideal way of managing multiple configs, added the changes